packages/babel-core/package.json

```diff
@@ -50,7 +50,7 @@
     "@babel/code-frame": "workspace:^",
     "@babel/generator": "workspace:^",
     "@babel/helper-compilation-targets": "workspace:^",
-    "@babel/helper-module-transforms": "condition:BABEL_8_BREAKING ? : workspace:^7.18.6",
+    "@babel/helper-module-transforms": "workspace:^",
```

Comment:
We don't, but `yarn-plugin-conditions` doesn't support ESM packages yet :grimacing: (because you need to statically know the imports/exports of the two packages, while it currently just does `module.exports = BABEL_8_BREAKING ? require("a") : require("b")`).

It's one of the TODOs I left in https://github.com/babel/babel/projects/16.